src/content/1.7/webservice/tutorials/creating-access.md

```diff
@@ -65,7 +65,7 @@ Creating API keys can be done with the class WebserviceKey.
 ```php
 <?php
 $apiAccess = new WebserviceKey();
-$apiAccess->key = 'GENERATE_A_COMPLEX_VALUE';
+$apiAccess->key = 'GENERATE_A_COMPLEX_VALUE_32_CHAR';
```

Comment:
```suggestion
$apiAccess->key = 'GENERATE_A_COMPLEX_VALUE_WITH_32_CHARACTERS';
```

Reply:
@PierreRambaud Now your example still isn't 32 chars but 43 chars.

Reply:
```32_CHARACTERS_COMPLEX_VALUE_HERE``` 

would be a better example imho.